elements/lisk-p2p/src/p2p.ts

```diff
@@ -897,6 +898,10 @@ export class P2P extends EventEmitter {
 			}
 		});
 
+		// According to LIP, add whitelist peers to triedPeer by upgrading them initially.
+		this._sanitizedPeerLists.whitelisted.forEach(whitelistPeer =>
```

Comment:
Is it possible to cover it by any integration test?

Reply:
I get your point but we don't want to test `new/triedPeer` in the integration tests. We already tested that when we do `upgradePeer` then it goes to `triedPeers` in peer_directory unit tests.